config/routes.rb

```diff
@@ -36,4 +38,10 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
-end
+
+  get "/admin/applications/:id", to: "admin/applications#show" 
+
+  get "/applications/new", to: "applications#new"
+  get "/applications/:id", to: "applications#show"
+  post "/applications", to: "applications#create"
```

Comment:
Per previous comments, breaking up controller actions and making more RESTful routes would lead to the addition of these new routes:
```
POST "applications/:application_id/pets", to "pet_applications#create"    # to add pet to application
PATCH "applications/:app_id/pets/:pet_id", to "pet_applications#update".     # to approve/reject pet
PATCH "applications/:id", to "applications#update"      # from user submitting an app to update status
```